README.md

```diff
@@ -52,3 +52,6 @@ Frequently asked questions:
   - A. In Windows, make sure to set the flag `-style fusion` at the end of the target path of the .exe
     - Example: `C:\Users\Administrator\Desktop\Supercell-Wx\bin\supercell-wx.exe -style fusion`
   - A. In Linux, if you're using KDE, Supercell Wx should automatically follow your theme settings.
+    
+- Q: How can I contribute?
+  - A. Head to [Developer Setup](https://supercell-wx.readthedocs.io/en/stable/development/developer-setup.html) to configure the supercell-wx development environment for your IDE. Currently Visual Studio and Visual Studio Code are supported, with support for other IDE's being added down the road
```

Comment:
"supercell-wx" -> "Supercell Wx"
"IDE's" -> "IDEs"
End with a period

Should also include a reference to CONTRIBUTING.md.

Reply:
Regarding "Support for other IDEs", other IDEs are expected to work, just not tested. This implies additional work would be needed to get things like CLion and Eclipse working, which I don't expect to be the case. Maybe "Visual Studio and Visual Studio Code are recommended", with other IDEs untested?

Reply:
How does this look?

`Currently Visual Studio and Visual Studio Code are recommended, with other IDEs remaining untested at this time.`

Wanting to make it as concise as possible

Reply:
Looks good to me!